### PR TITLE
Add changelog text to output

### PR DESCRIPTION
**Pull request by mfeifer**
Adds the string used in the changelog file to the output.

[skip ci]

### CODE REVIEW

**Comment by mfeifer** [comment]
@bdunne I need to run this same process on a few different repos. I was considering also defining a "repo" variable in each rb file to just make it so that I don't have to type out the repo name because I use it more than once.

File: closed_issues.rb [hunk 1]
```diff
@@ -7,8 +7,9 @@
 
 File.open('closed_issues_master_repo.csv', 'w') do |f|
     f.puts "Milestone Statistics for: #{prs.first.milestone.title}"
-    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS"
+    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS,CHANGELOGTEXT"
     prs.each do |i|
-    	f.puts "#{i.number},#{i.title},#{i.user.login},#{i.assignee && i.assignee.login},#{i.labels.collect(&:name).join(" ")}"
+    	i.changelog = "#{i.title} #[#{i.number}](https://github.com/ManageIQ/manageiq/pull/#{i.number})"
```
Comment (bdunne) [inline closed_issues.rb]: `i.changelog = "#{i.title} #[#{i.number}](#{i.pull_request.html_url})"`

Comment (mfeifer) [inline closed_issues.rb]: so, I made things harder on myself.

👍 

at least I got the concatenation right!

File: closed_issues.rb [hunk 2]
```diff
@@ -7,8 +7,9 @@
 
 File.open('closed_issues_master_repo.csv', 'w') do |f|
     f.puts "Milestone Statistics for: #{prs.first.milestone.title}"
-    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS"
+    f.puts "NUMBER,TITLE,AUTHOR,ASSIGNEE,LABELS,CHANGELOGTEXT"
```
Comment (bdunne) [inline closed_issues.rb]: This doesn't match the line below.  You added `closed_at` before `changelog`

Comment (mfeifer) [inline closed_issues.rb]: 👍 

reading is hard.

**Pull request merged**